Gabriele/comparisonOp.js

```diff
@@ -0,0 +1,77 @@
+// 0. Declare two variables "apples" and "oranges". Assign a value of 20 to apples. Assign a value of 30 to oranges.
+
+let apples = 20;
+let oranges = 30;
+console.log(apples, oranges);
+
+// 1. Check whether apples and oranges are equal.
+
+console.log("is the value of apple and oranges equal?");
+console.log(apples === oranges); // false
+
+
+// .2 Check whether apples and oranges are not equal.
+
+console.log("are apples and oranges not equal?");
+console.log(apples !== oranges); // true
+
+// .3 Check whether apples is greater than oranges.
+
+console.log("is apples greater than oranges?");
+console.log(apples > oranges); // false
+
+// 4. Check whether apples is less than or equal to oranges.
+
+console.log("is apples less or equal to oranges?");
+console.log(apples <= oranges); // true
+
+// 5. Check whether oranges is greater than apples.
+
+console.log("is oranges greater than apples?");
+console.log(oranges > apples); // true
+
+// 6. Declare another variable "mangoes" and give it a value of 5. Multiply mangoes and apples, and check whether this result is greater than mangoes added to oranges.
+
+let mangoes = 5;
+console.log(mangoes * apples); // 100
+let result = 100;
```

Comment:
we can store the let result = mangoes * apples;

and the same thing with the rest